tests/layer_tests/jax_tests/test_add.py

```diff
@@ -0,0 +1,124 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+
+from jax_layer_test_class import JaxLayerTest
+import jax.numpy as jnp
+
+class TestAdd(JaxLayerTest):
+    def _prepare_input(self):
+        lhs = jnp.array(np.random.randint(-10, 10, self.lhs_shape).astype(self.lhs_type))
+        rhs = jnp.array(np.random.randint(-10, 10, self.rhs_shape).astype(self.rhs_type))
```

Comment:
let us differentiate data generation for floating-point type and integer type.
For floating-point type, we want to have data values like 1.5 and 2.1